plugins/devo/help.md

```diff
@@ -0,0 +1,160 @@
+# Description
+
+[Devo](https://www.devo.com/) is the cloud-native logging and security analytics solution that delivers real-time visibility for security and operations teams
+
+# Key Features
+
+* Query for log results
+* Trigger workflows on new alerts
+
+# Requirements
+
+* A Devo authentication token
+
+The Devo authentication token must have access to the tables you are building your query for. In additon
```

Comment:
```suggestion
The Devo authentication token must have access to the tables you are building your query for. In addition
```

Reply:
I don't think we can put text in here, as it won't get rendered in the Ext Lib. I think this happened last month in another integration that you were working on @jmcadams-r7 

It's bulleted list only. Any additional need to go in the Connection  section